packages/clarity-js/src/interaction/pointer.ts

```diff
@@ -11,6 +11,8 @@ import encode from "./encode";
 
 export let state: PointerState[] = [];
 let timeout: number = null;
+let activeTouchPointId = 0;
+let activeTouchPointIds = new Set<number>();
```

Comment:
nit: this can be `const`

Reply:
the value changes right? how can it be const?

Reply:
const for a set means that the pointer doesn't change essentially, it doens't mean the contents of the set are the same (like an array you can have const x = []; x.push(1); x.push(2); //you just can't then do x = [2,3,4]

Reply:
changed to const